boards/arduino-mkr1000/dist/openocd.cfg

```diff
@@ -0,0 +1,7 @@
+source [find interface/cmsis-dap.cfg]
```

Comment:
Does this board support being flashed using openocd through CMSIS-DAP? If so, this option is not usable as there is no option the board's Makefile.include... So I'd say either remove this file or add the needed options to the Makefile.include